CONTRIBUTING.md

```diff
@@ -1,41 +1,75 @@
-# Contributing to `cosign`
+# Contributing to Cosign
 
-The following document will detail the steps needed in order to ensure all of the correct processes are followed when contributing to `cosign`.
+Thank you for considering contributing to Consign! We welcome any contributions, whether it's bug fixes, new features, or improvements to the existing codebase.
 
-The following are needed to be completed before a Pull Request is raised:
+## Your First Pull Request
 
-- unit tests written, run, and passing
-- lint run
-- document generation run
+Working on your first Pull Request? You can learn how from this free video series:
 
-## Building
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
Can we omit? Instead, I'd prefer to link to https://github.com/sigstore/community/blob/main/CONTRIBUTING.md, which includes some high level information on how to contribute to Sigstore projects.